CMGTools/MonoXAnalysis/python/tools/eventVars_monojet.py

```diff
@@ -3,8 +3,8 @@
 from CMGTools.MonoXAnalysis.tools.BTagWeightCalculator import BTagWeightCalculator
```

Comment:
Yes, this is a duplicate of a module that @gpetruc gave me, and it should be an updated version from H->bb of this file : 
 https://github.com/CERN-PH-CMG/cmg-cmssw/blob/ea63f1bcfbf798253aa97fd8814ce7871b83d801/PhysicsTools/Heppy/python/physicsutils/BTagWeightCalculator.py

if experts of that can comment, we could update that file in heppy and I will just import from there.